netbox/circuits/forms.py

```diff
@@ -265,8 +265,10 @@ class Meta:
         ]
 
 
-class CircuitFilterForm(BootstrapMixin, CustomFieldFilterForm):
+class CircuitFilterForm(BootstrapMixin, TenancyFilterForm, CustomFieldFilterForm):
     model = Circuit
+    # Order the form fields, fields not listed are appended
+    field_order = ['q', 'type', 'provider', 'status']
```

Comment:
Where did `field_order` come from? AFAIK it isn't a thing. Fields can be listed under `Meta` as `fields` but by default I believe they appear in order of declaration on the model.

Reply:
field_order was added in DJango 1.9

It lets you order your form fields in a certain manner.

https://docs.djangoproject.com/en/2.1/ref/forms/api/#notes-on-field-ordering

This was done because if you don't do it, tenant and tenant_group appear at the top of the search filter.